src/stories/Library/Modals/modal-pause/ModalPause.tsx

```diff
@@ -29,33 +29,29 @@ export const ModalPause: React.FC<ModalPauseProps> = ({
       <div className="modal-pause__dropdowns mt-24">
         <div className="datepickers">
           <div className="datepicker">
-            <span className="datepicker-toggle">
-              <label htmlFor="startDate" className="text-body-medium-regular">
-                Startdato
-              </label>
-              <input
-                type="date"
-                name="startDate"
-                id="startDate"
-                className="datepicker-input"
-                aria-label="startDate"
-              />
-            </span>
+            <label htmlFor="startDate" className="text-body-medium-regular">
+              Startdato
+            </label>
+            <input
+              type="date"
+              name="startDate"
+              id="startDate"
+              className="datepicker-input"
+              aria-label="startDate"
+            />
           </div>
 
           <div className="datepicker">
```

Comment:
This is is not really related to this PR. But wouldn't be an idea to make the datepicker a generic design component whit its own story? Isn't it a building block that could be used in other places besides the modal pause scope?